draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1224,13 +1222,12 @@ Inputs:
 Output: u, an element in F.
 
 Steps:
-1. m' = H(msg) || I2OSP(ctr, 1)
-2. for i in (1, ..., m):
-3.   t = ""     // initialize t to the empty string
-4.   for j in (1, ..., W):
-5.     t = t || H( m' || I2OSP(i, 1) || I2OSP(j, 1) )
-6.   e_i = OS2IP(t) mod p
-7. return u = ( e_1, ..., e_m )
+1. L = ceil((ceil(log2(p)) + k) / 8)
+2. m' = H(msg) || I2OSP(ctr, 1)
+3. for i in (1, ..., m):
+4.   t = HKDF-Expand(m', I2OSP(i, 1), L)
```

Comment:
```suggestion
4.   t = HKDF-Expand-H(m', I2OSP(i, 1), L)
```

Reply:
Hmm, this isn't common notation that I'm familiar with. Typically H is inferred from context, which I think it is here.

Reply:
(Read your suggestion below, and provided it's defined as a parameter then this works great. Will adopt it.)